src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -464,20 +464,19 @@ public void run() {
                                             } catch (Exception e) {
                                                 // TODO design error handling reporting
                                                 if (Utilities.isDebugMode()) {
-                                                    System.err.println("Exception submitting run for: " + event);
+                                                    System.err.println("Exception submitting run for event: " + event + ": " + e);
                                                 }
-                                                log.log(Level.WARNING, "Exception submitting run for: \\" + event);
-                                                
+                                                log.log(Level.WARNING, "Exception submitting run for event: " + event + ": ", e);
```

Comment:
Missing exception message in log message, the message will show this when after the : should be a message
Exception submitting run for event: <EVENT>:   
before:
+ event + ": ", e);
change to:
+ event + ": ", + e, e);

Reply:
I don't think this is a good suggestion.  I believe that the form  `log.log(LEVEL,String, Throwable)` (which is what the new code does) already puts the exception in the log (that's what the `Throwable` parameter is for).  Your suggested change would cause the exception to be placed in the log TWICE.

And anyway, the suggestion to change it to `event + ": ", + e, e);` is going to generate a syntax error -- there are too many commas in that suggested change.  But I think you meant "change to: `event + ": " + e, e);` " (which would put the exception in the log twice).